src/library/scala/concurrent/Future.scala

```diff
@@ -20,11 +20,16 @@ import scala.collection.generic.CanBuildFrom
 import scala.reflect.ClassTag
 
 
-/** The trait that represents futures.
+/** 
  *
- *  Asynchronous computations that yield futures are created with the `Future.apply` call:
+ *  A `Future` represents a value which may or may not *currently* be available,
```

Comment:
This scaladoc is inconsistent (in style) with the others, fix this or the others?